client/my-sites/controller.js

```diff
@@ -373,14 +374,20 @@ export function siteSelection( context, next ) {
 		const state = getState();
 		const isAtomicSite = isSiteAutomatedTransfer( state, siteId );
 		const userCanManagePlugins = canCurrentUser( state, siteId, 'manage_options' );
-		const calypsoify = isAtomicSite && config.isEnabled( 'calypsoify/plugins' );
+		const calypsoifyPlugins = isAtomicSite && config.isEnabled( 'calypsoify/plugins' );
```

Comment:
```suggestion
		const calypsoifyPlugins = config.isEnabled( 'calypsoify/plugins' );
```

I think we shouldn't have an `isAtomicSite` check here, since we want to enable this for Simple sites now too? And `calypsoifyPlugins` is part of the redirect check bellow too.